public/angular-ui-router.min.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
Use bower to manage angular dependencies. Use bower.json. Do not commit angular files to github repo.